frontend/src/components/Playback/Playback.js

```diff
@@ -105,7 +104,11 @@ const Playback = ({
                         pauseAudio(playConfig);
                         return;
                     }
-                    let latency = playAudio(playConfig, sections[index]);
+                    // if the current player has resume_play set to true,
+                    // retrieve previous player's decisionTime from sessionStorage
+                    const playheadShift = playConfig.resume_play ? 
+                        parseFloat(window.sessionStorage.getItem('decisionTime')) : 0;
+                    let latency = playAudio(playConfig, sections[index], playheadShift);
```

Comment:
Additionally, I think it might be nice to combine this whole bit in a separate function named something like `startOrContinuePlayAudio`, e.g. something that explains what is happening. This whole bit means it continues the audio from where the participant left off, if it actually left off, right?